lib/addons/flagger/index.ts

```diff
@@ -0,0 +1,72 @@
+import 'source-map-support/register';
+import * as blueprints from '../../../lib';
+import { Construct } from 'constructs';
+import { Values } from "../../spi";
+import merge from "ts-deepmerge";
+
+/**
+ * User provided options for the FlaggerAddonProps values.
+ */
+export interface FlaggerAddOnProps extends blueprints.HelmAddOnUserProps {//this is the root level
```

Comment:
please remove the comment // this is the root level 
it is evident

Reply:
That was a personal note I forgot to remove, thanks for the catch.